dockerfile_scripts/install_deepspeed.sh

```diff
@@ -5,17 +5,23 @@ set -e
 apt-get update
 DEBIAN_FRONTEND=noninteractive apt-get install -y pdsh libaio-dev
 
-#Older versions of deepspeed require pinned pydantic version
-python -m pip install pydantic==1.10.11
-
-#Precompile supported deepspeed ops except sparse_attn
-export DS_BUILD_OPS=1
+# Install some dependencies for the LLM test
+pip install accelerate==0.22.0 arrow==1.2.3 datasets==2.14.5 huggingface-hub==0.17.3 packaging==23.1 safetensors==0.3.3 setuptools==65.7.0 tokenizers==0.14.1 transformers==4.34.1 xxhash==3.3.0 evaluate
+#Precompile deepspeed ops except sparse_attn which has dubious support
+# Skip precompiling since this fails when using the NGC base image.
+# Need to verify that DS can use NCCL correctly for the comms, etc.
+#export DS_BUILD_OPS=1
 export DS_BUILD_SPARSE_ATTN=0
 export DS_BUILD_EVOFORMER_ATTN=0
-export DS_BUILD_CUTLASS_OPS=0
-export DS_BUILD_RAGGED_DEVICE_OPS=0
+cuda_ver_str=`echo $CUDA_VERSION | awk -F "." '{print $1"."$2}'`
+#export CUDA_DIR="/usr/local/cuda-$cuda_ver_str/targets/sbsa-linux"
+#export CUDA_HOME="/usr/local/cuda-$cuda_ver_str/targets/sbsa-linux"
+#export CUDA_DIR="/usr/local/cuda-$cuda_ver_str/targets/sbsa-linux"
```

Comment:
The targets subdir will be dependent on the platform. For aarch64 it will be the sbsa-linux (ie, GH200 nodes). Should determine the subdir programmatically here to make it work for both aarch64 and x86_64.